kuma/landing/views.py

```diff
@@ -10,6 +11,7 @@
 from kuma.search.serializers import GroupWithFiltersSerializer
 
 
+@cache_page(HOMEPAGE_CACHE_TIMEOUT, key_prefix=HOMEPAGE_CACHE_PREFIX)
```

Comment:
That'll need to be:

    @cache_page(settings.HOMEPAGE_CACHE_TIMEOUT,
                key_prefix=settings.HOMEPAGE_CACHE_PREFIX)
    def home(request):
    # ...